### ISSUE TITLE
Issue running prest binary

### ISSUE THREAD

**Issue by BDive** [prest/prest#844]
Hi, I downloaded the latest binary for windows, prest version 1.4.0, when running it, I am getting the following message. 
- the URL per message, https://docs.prestd.com/prestd/deployment/server-configuration does not exist also

```
You are using v1 of prestd configs, please migrate to v2.

v1 will be deprecated soon.

View more at https://docs.prestd.com/prestd/deployment/server-configuration←[0;00m
←[93m2023/12/04 09:28:56 [warning] adapter is not set. Using the default (postgres)←[0;00m
←[93m2023/12/04 09:28:56 [warning] You are running prestd in public mode.←[0;00m
```
 My config
prest.toml
```
migrations = "./migrations"

# debug = true
# enabling debug mode will disable JWT authorization

[http]
port = 3000

[jwt]
key = "secret"
algo = "HS256"

[auth]
enabled = true
type = "body"
encrypt = "MD5"
table = "users"
username = "username"
password = "password"

[pg]
host = "127.0.0.1"
user = "postgres"
pass = "postgres"
port = 5432
database = "resto"
single = true
## or used cloud factor
# URL = "postgresql://user:pass@localhost/mydatabase/?sslmode=disable"

[ssl]
mode = "disable"
sslcert = "./PATH"
sslkey = "./PATH"
sslrootcert = "./PATH"

[expose]
enabled = true
databases = false
schemas = false
tables = true
```

**Comment by arxdsilva** [comment]
Hi @BDive, thanks for contacting, we forgot to update this link message, it should be only a warning message, could you double check that it works overall for you?

I'll make a update patch with the right link

**Comment by arxdsilva** [comment]
@BDive for now we dont have user management, so your authenticated user will have all the permissions set on the toml/envs, in the future we want to grant that but I cant promise any timeline on that.

**Comment by BDive** [comment]
Thanks @arxdsilva, please consider loading the config from database so it can be defined dynamically via database/SQL query

**Comment by arxdsilva** [comment]
@BDive released `v1.4.1` to solve this visual bug
